apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.sln

```diff
@@ -105,7 +105,6 @@ Global
 		{97EC9E30-ADD3-4D80-A2D5-8D611557813E}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{97EC9E30-ADD3-4D80-A2D5-8D611557813E}.Debug|x86.Build.0 = Debug|Any CPU
 		{97EC9E30-ADD3-4D80-A2D5-8D611557813E}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{97EC9E30-ADD3-4D80-A2D5-8D611557813E}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
I don't understand this change. I can see why we wouldn't want to *run* the generator, but why wouldn't we want to even compile it?

Reply:
The solution wasn't building on travis, so this was a bit of a hack, but here was my thinking: when debugging, you are likely to have the solution open in VS with the Debug configuration selected. In that case, you want the project to build with the solution. However, when the solutions are building in Release config on the CI machines, there's no need to build it. When running the project directly from generateapis.sh, it will get built automatically before running.